drivers/node/ssh/ssh.go

```diff
@@ -88,14 +87,14 @@ func useSSH() bool {
 }
 
 // Init initializes SSH node driver
-func (s *SSH) Init() error {
+func (s *SSH) Init(nodeOpts node.InitOptions) error {
 
 	nodes := node.GetWorkerNodes()
 	var err error
 	if useSSH() {
 		err = s.initSSH()
 	} else {
-		err = s.initExecPod()
+		err = s.initExecPod(nodeOpts.SpecDir)
```

Comment:
save SpecDir parameter in *SSH struct, so you don't need to pass it to initExecPod function

Reply:
Done